tests/integration/nm/linux_bridge_test.py

```diff
@@ -101,22 +123,47 @@ def _bridge_interface(state):
             _delete_iface(p[LB.PORT_NAME])
 
 
+@mainloop_run
+def _modify_bridge(bridge_desired_state):
+    bridge_config = bridge_desired_state[LB.CONFIG_SUBTREE]
+    _modify_bridge_options(bridge_config)
+    _modify_ports(bridge_config[LB.PORT_SUBTREE])
+
+
+def _modify_bridge_options(bridge_config):
+    br_options = bridge_config.get(LB.OPTIONS_SUBTREE)
+    nmdev = nm.device.get_device_by_name(BRIDGE0)
+    conn = nm.connection.ConnectionProfile()
+    conn.import_by_id(BRIDGE0)
+    iface_bridge_settings = _create_iface_bridge_settings(br_options, conn)
+    new_conn = nm.connection.ConnectionProfile()
+    new_conn.create(iface_bridge_settings)
+    conn.update(new_conn)
+    conn.commit(save_to_disk=False, nmdev=nmdev)
```

Comment:
why don't we want this to persist  ?

Reply:
Because we do not need it. We are not testing NM persistent capabilities here.